release/go-licenses/Dockerfile

```diff
@@ -16,8 +16,11 @@
 # limitations under the License.
 ###############################################################################
 
-FROM golang:1.16-bullseye
-RUN go get github.com/google/go-licenses@v1.0.0
+FROM golang:1.18-bullseye
+
+WORKDIR /usr/src/app
+COPY go.mod ./
+RUN go install github.com/google/go-licenses@v1.0.0
```

Comment:
We should file an issue to bump the version to the [latest version (1.2.1)](https://pkg.go.dev/github.com/google/go-licenses@v1.2.1). Since we're trying to unblock things, we're probably fine not doing it in this PR.

I'd consider just making the target @latest instead though, it's unlikely to cause breaking changes, and allow additional license detection over time.
https://github.com/google/go-licenses/tags

Reply:
I agree that latest is probably best, I just wanted to keep this change minimal since this was blocking others. #21901 bumps this to latest.

If we run into issues with that we can also pin the version at a later date.